src/components/settings/cameras/CameraConfigDialog.vue

```diff
@@ -148,16 +148,17 @@
 
         <app-setting
           v-if="camera.service === 'iframe'"
-          :title="$t('app.setting.label.height')"
+          :title="$t('app.setting.label.aspect_ratio')"
+          :sub-title="`[${$t('app.setting.label.aspect_ratio_format')}]`"
```

Comment:
Why not put the whole aspect ratio string as part of the translation instead of having this formatting here?